src/features/movies/MoviePage/MovieDetailsTile/index.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import {
+  DetailsTile,
+  Image,
+  ProductionYear,
+  Text,
+  Birth as ProductionAndRelease,
+  MobileDescription,
+  DesktopDescription,
+  DesktopGrayText,
+  Frame,
+  Name as Title,
+  Tags,
+  Tag,
+  MovieRatingImg,
+  MovieRatingText,
+  MovieRatingVotes,
+  Rating,
+  DesktopMovieRatingVotes,
+} from "./styled";
+import star from "../../MovieTile/ratingStar.svg";
+
+
+const MovieDetailsTile = ({ movie }) => (
+  <DetailsTile>
+    <Image src={`https://image.tmdb.org/t/p/w342${movie.poster_path}`} alt="" />
+    <section>
+      <Title>{movie.original_title}</Title>
+      <ProductionYear>{movie.release_date.slice(0, 4)}</ProductionYear>
```

Comment:
tutaj również może się pojawić problem przy null'u/undefined:
{movie.release_date ? movie.release_date.slice(0,4) : "????"}
Ja to rozwiązałem w taki sposób, myślę, że można to tak samo tutaj przekleić

Reply:
musimy rozpoznać temat jak to jest, czy rzeczywiście może nie być tej release date, czy to z innego powodu wywalało błędy, bo w sumie nie powinno tak być, że trzeba sprawdzać każdą informację z api, czy istnieje, bo to by było słabe api :P